### ISSUE TITLE
`TypeError` on `Fraction` + `math.log`

### ISSUE THREAD

**Issue by tybug** [pschanely/hypothesis-crosshair#14]
https://crosshair-web.org/?crosshair=0.1&python=3.8&gist=53cb15615819ffd4e29fe0c29b1cfe38

this may not be the smallest counterexample, but it was as small as I could easily make it. `min(n, 1)` is to avoid crosshair finding domain errors.

This is the root cause of hypothesis-crosshair erroring on this property:

```python
@given(st.decimals(min_value=1.0, max_value=1.5))
def f(n):
    pass
f()
```

**Comment by tybug** [comment]
gah, meant to file this against https://github.com/pschanely/crosshair. oh well!

**Comment by pschanely** [comment]
Thanks for the report! Fixed in [0.0.59](https://pypi.org/project/crosshair-tool/0.0.59/)!

**Comment by tybug** [comment]
ah, here's a follow-up error for the same strategy: https://crosshair-web.org/?crosshair=0.1&python=3.8&gist=449b64ef5fa19a218fa41da5535eb66a
